Assets/BidMachine/Api/AuctionResult.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace BidMachineAds.Unity.Api
+{
+    [Serializable]
+    public sealed class AuctionResult
+    {
+        public string DealID { get; set; }
+        public string DemandSource { get; set; }
+        public string CID { get; set; }
+
+        public CustomParams CustomParams { get; set; }
+        public CustomExtras CustomExtras { get; set; }
```

Comment:
Android does not have `CustomExtras`, only `NetworkParams` with type `Dictionary<string, string>`.

Reply:
Is it the same?